src/App.tsx

```diff
@@ -0,0 +1,187 @@
+        footer: {
+            backgroundColor: Colors.black[500],
+            padding: theme.spacing(2),
+            color: Colors.white[50],
+            flexShrink: 0,
+            textAlign: 'center',
+        },
+        FAB: {
+            position: 'fixed',
+            right: theme.spacing(3),
+            bottom: theme.spacing(3),
+            boxShadow: theme.shadows[10],
+        },
+    })
+);
+
+function App(): JSX.Element {
+    const theme = useTheme();
+    const classes = useStyles(theme);
+    const matchesSM = useMediaQuery(theme.breakpoints.down('sm'));
+    const matchesXS = useMediaQuery(theme.breakpoints.down('xs'));
+    return (
+        <div className={classes.root}>
+            <div className={classes.content}>
+                <div className={classes.ellipseBackground} />
+                <div
+                    className={classes.title}
+                    style={matchesXS ? { backgroundImage: `url(${titleImageSM})`, width: '70%' } : undefined}
+                />
+                <div
+                    className={classes.SimpleCardContainer}
+                    style={matchesSM ? { flexDirection: 'column', alignItems: 'center' } : undefined}
+                >
+                    <SimpleCard
+                        image={angularImage}
+                        title={'Angular / Ionic'}
+                        body={
+                            'Our Angular component library is intended for use in applications built using Angular and Angular Material. They can also be used in Ionic applications that are built with Angular. '
+                        }
+                        packageName={'@pxblue/angular-components'}
+                        url={'/angular'}
+                    />
+                    <SimpleCard
+                        image={reactImage}
+                        title={'React'}
+                        body={
+                            'Our React component library can be used for applications built using React v16+. They include strong type definitions so they will work well in either JavaScript or TypeScript projects.'
+                        }
+                        packageName={'@pxblue/react-components'}
+                        url={'/react'}
+                    />
+                    <SimpleCard
+                        image={reactNativeImage}
+                        title={'React Native'}
+                        body={
+                            'Our React Native component library can be used for mobile applications built with React Native v0.60+. They include strong type definitions so they will work well in either JavaScript or TypeScript projects.'
+                        }
+                        packageName={'@pxblue/react-native-components'}
+                        url={'/react-native'}
+                    />
+                </div>
+                <div className={classes.section}>
+                    <div className={classes.sectionContentContainer}>
```

Comment:
Could we make these 'sections' into a reusable component?

Reply:
Resolve: after discussion teams, we realized this may not be the best time for it